jupyterhub/jupyterhub/overlays/trusted-ca-bundle-path/kustomization.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+commonLabels:
+  app.kubernetes.io/part-of: jupyterhub
+
+bases:
+  - ../../base
+resources:
+  - trusted-cabundle-configmap.yaml
+patchesJson6902:
```

Comment:
this notation should make sure the ops "add" and replace with regards to json paths work for the two configmaps in base. In the past, I only worked with patchesStrategicMerge yaml-based, but this approach seems cool, too.